wagtail/api/v2/tests/test_pages.py

```diff
@@ -941,6 +941,22 @@ def test_ordering_by_unknown_field_gives_error(self):
             content, {"message": "cannot order by 'not_a_field' (unknown field)"}
```

Comment:
Here's a test that checks for an error, could use a similar approach for testing your other error.